mvpa/cosmo_crossvalidate.m

```diff
@@ -16,6 +16,14 @@
 %                       first or second dimension of ds. Normalization
 %                       parameters are estimated using the training data
 %                       and applied to the testing data.
+%     .pca              optional, (default: false) Set to true
```

Comment:
The case I can think of is to use pca just for the whitening and not for dimensionality reduction. But this edge-case could also be achieved by using pca_explained_ratio=1;